NEWS

```diff
@@ -11,7 +11,7 @@
 ### Features:
 ### Bugfixes:
 
-## 1.13.0 (May 27, 2022)
+## 1.13.0 (June 26, 2022)
```

Comment:
it should be 1.13.0-rc2

Reply:
> 1.13.0-rc2

should we then also update extra version in configure.ac?

Reply:
yes, let's try this, it would be first time we use it in a release.

Reply:
we don't have it in the NEWS now, because it was replaced it to v1.12.0 (for example) before the GA

Reply:
this is how it was done so far, so the diff reflects the changes vs. the previous GA.
@shamisp WDYT?

Reply:
We do this to track in case somebody picks up one of the RC instead of releases - this way it is actually reflected in news.

Reply:
OK, added section on Wiki page https://github.com/openucx/ucx/wiki/Release-process#release-candidates